support-frontend/assets/helpers/render.js

```diff
@@ -6,7 +6,7 @@ import { logException } from 'helpers/logger';
 // Without this the build-time pre-rendering breaks, because fetch is undefined when running with node
 const safeFetch = (url: string, opts) => {
   if (typeof fetch !== 'undefined') {
-    fetch(url, opts)
+    fetch(url, opts);
```

Comment:
oh I got these changes in my PR as well, I reverted them as I thought the formatter had gone funny

Reply:
It's when you run `yarn validate` which in turn runs `lint --fix`, that auto fixes any errors it can find.